index.html

```diff
@@ -83,7 +83,7 @@ <h1 slot="main-title">
                 </div>
             </noscript>
 
-            <div slot="about" style="margin-top: 40px; margin-left: 10px">
+            <div slot="about" id="about" style="margin-top: 40px; margin-left: 10px">
                 <h2 class="text-primary pt-5 mb-5">
```

Comment:
```suggestion
            <div slot="about" style="margin-top: 40px; margin-left: 10px">
                <h2 id="apropos" class="text-primary pt-5 mb-5">
```

Je ne suis pas sûr d'avoir compris le cas d'usage (même via les apps mobiles). Mais comme on le fait pour l'autre grosse section de la page (en-savoir-plus), j'ai ajouté l'id directement sur le titre de la section concernée. Cela évite les problèmes de shadowroot et cela fonctionne bien.

Reply:
J'ai également mis l'ancre en français.